docker/deploy

```diff
@@ -6,6 +6,9 @@ dfx start --background --quiet
 
 ./docker/wait-port "$DFX_PORT"
 
+dfx canister create internet_identity --specified-id rdmx6-jaaaa-aaaaa-aaadq-cai
```

Comment:
Can you please add a to-do here saying that it is a temporary solution? For what I understood from Max, it will be solved by ic-terraform, even if it is not something so in the short-term. Right, @bitdivine ?

Reply:
I have to temper your expectations.  `ic-terraform` is in very early stages of development and it will be months until we can use it extensively.  Maybe even years if we are kept very busy, with no time to improve the infrastructure.  But the sole purpose of `terraform` is to map the dependencies between infrastructure components and deploy them in the right order.  The above is an example of having to fulfil such a  dependency so, yes, that should be covered by `terraform`.  I am looking forward to it.

Reply:
Added the comment. And thanks for the explanation 👍